### ISSUE TITLE
Allow permanent sharing with another device / person

### ISSUE THREAD

**Issue by fschlich** [SecUSo/privacy-friendly-shopping-list#14]
I need to coordinate shopping with my partner. This means keeping several lists in sync, instantly, permanently, crossing off items simultaneously while visiting different shops. Or,  more frequently, each of us adding items when we realize something is needed, and in the end one of us is going shopping and already has everything needed on the phone.

The app we're currently using needed access to a Gmail account, presumably for message passing /  list storage. Coming up with a lightweight and easy-to-use solution for this is probably the hardest part...

**Comment by alleyoopster** [comment]
I am also keen to see this feature added.

**Comment by himbeere** [comment]
Yeah. I need that feature too. Currently i am using https://f-droid.org/repository/browse/?fdfilter=it&fdid=org.janb.shoppinglist&fdpage=16&fdstyle=grid which stores the list items in a mysql db. Good enough for me. Sadly this app seems te be abandoned.

**Comment by supertobi** [comment]
@ghost You link no longer woks.
@Yonjuni Any news about this feature? Is anything planned?

**Comment by petermayer** [comment]
@supertobi We are looking into integrating Import/Export/Synchronisation of the data of several of our apps. Once the details of this are fixed, it might affect this issue as well. Hang in there, you'll hear it here, once there is something to report.
